haiku2048.rdef

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2015 Markus Himmel
+ * Distributed under the terms of the MIT License
+ * Haiku Icon by Stephanie Fu
+ */
+
+resource app_signature "application/x-vnd.twofx.haiku2048";
```

Comment:
The app signature of this program is declared as `application/x-vnd.Haiku-Haiku2048` in App.cpp. One of the two or both should be changed so they match; I don't know what naming conventions there are for app signatures, they should be changed to something idiomatic.